homeassistant/components/ozw/cover.py

```diff
@@ -52,6 +57,12 @@ def percent_to_zwave_position(value):
 class ZWaveCoverEntity(ZWaveDeviceEntity, CoverEntity):
     """Representation of a Z-Wave Cover device."""
 
+    def __init__(self, values):
+        """Initialize Z-Wave cover device entity."""
+        self._open_button_pressed = False
+        self._close_button_pressed = False
+        super().__init__(values)
+
     @property
     def supported_features(self):
```

Comment:
We can remove this, since the default cover entity support now matches the support we need.

Reply:
Nice catch! Removed everything related.